test/e2e/storage/volume_metrics.go

```diff
@@ -100,7 +100,7 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 		backoff := wait.Backoff{
 			Duration: 10 * time.Second,
 			Factor:   1.2,
-			Steps:    3,
+			Steps:    8,
```

Comment:
this gives ~40s for detach, is it enough to stop flakes? ebs is given max 21 steps.

Reply:
isn't that ~80s @rootfs ?

Reply:
Actually because it is exponentially backing off the actual time given is 260 seconds.

Reply:
plugging these numbers into `ExponentialBackoff`, the max duration is 36s, entire duration is 154s. This should be sufficient for most cases. But since test flakes are at the mercy of cloud providers, let's sync with the same [max steps in ebs](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go#L157)

Reply:
ack done.